app/addons/components/assets/less/loading-lines.less

```diff
@@ -0,0 +1,65 @@
+@height: 30px;
+@margin: 3px;
+@width: 10px;
```

Comment:
suggestion: more specific name to avoid collisions: e.g. 

```
@loadingLinesHeight: 30px;
@loadingLinesMargin: 3px;
@loadingLinesWidth: 10px;
```